marblerun/workflows/define-manifest.md

```diff
@@ -142,19 +140,26 @@ The general format is the following:
 
 `{{ <encoding> <name of secret> }}`
 
-The following enconding types are available.
+The following enconding types are available to `Files` and `Env`:
 
-* `raw`: raw bytes
 * `hex`: hex string
 * `base64`: Base64 encoding
 * `pem`: PEM encoding with a header matching the type of the requested key or certificate
 
+
+The following encoding types are only available to `Files`:
+
+* `raw`: raw bytes
+
+The following encoding types are only available to `Env`:
+
+* `string`: similar to `raw`, but does not allow [NULL bytes](https://man7.org/linux/man-pages/man7/environ.7.html). Since the content in non-user-defined secrets is random, and can't be controlled, only secrets with `UserDefined=true` are allowed to use this encoding
```

Comment:
```suggestion
* `string`: similar to `raw`, but does not allow [NULL bytes](https://man7.org/linux/man-pages/man7/environ.7.html). Since the content in non-user-defined secrets is random, and cannot be controlled, only secrets with `UserDefined` set to `true` are allowed to use this encoding.
```